README.md

```diff
@@ -145,6 +145,13 @@ const connectionParameters = sdk.ConnectionParameters.ofUserAndPassword(
                                     "https://myInstance.campaign.adobe.com", 
                                     "admin", "==ims_service_token_here");
 ```
+## Login with IMS access token
```

Comment:
* In read me, and code comments, can you specify that we are talking about user tokens here? Unless you tell me it also works with technical tokens. It that the case, indicate that it works with both.
* Also how do we get this bearer token? Can we add a small paragraph on this? I know it’s not done with Campaign, but it”s good to help users to get started quickly